MiniApp/Classes/core/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -87,7 +87,11 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
         case .getPoints:
             fetchPoints(with: callbackId)
         case .getHostEnvironmentInfo:
-            getHostEnvironmentInfo(with: callbackId)
+            if hostAppMessageDelegate?.getEnvironmentInfo != nil {
+                getEnvironmentInfo(with: callbackId)
+            } else {
+                getHostEnvironmentInfo(with: callbackId)
+            }
```

Comment:
This else block will never get called right? 
`getHostEnvironmentInfo` is deprecated also now in this PR we have extension for `getEnvironmentInfo` ?

Reply:
@rleojoseph I was unsure if we should keep the support for the old delegate method because it might have been implemented so shouldn't the deprecated implementation work until we remove it with the release after?

Reply:
should be good now 💥